### PR TITLE
Adding a paragraph to state other logging options

### PR DESCRIPTION
**Pull request by fwji**

### CODE REVIEW

**Comment by fwji** [comment]
@t-koman Hi Tom, can you do an ID review of this change.

**Comment by t-koman** [comment]
@fwji Frank, I reviewed/edited the additional paragraphs and attached my revisions.   Let me know if you have any questions....
[aggregation_tool.docx](https://github.com/kabanero-io/draft-guide-app-logging/files/3796265/aggregation_tool.docx)

**Comment by fwji** [comment]
@t-koman Thanks for the review, I'd like to keep this line tho
**_There are a few log aggregation products available on the market that offer similar capabilities._**

File: README.adoc [hunk 1]
```diff
@@ -14,9 +14,9 @@ permalink: /guides/app-logging/
...
+A Kubernetes environment constantly produce logs by its running pod processes. To effectively manage this log data and ensure no loss of log data occurs when a pod terminates, deploy a a log aggregation tool on the user's OKD cluster to control the logging activities. Log aggregation tools help users persist, search, and visualize the log data that is gathered from the pods across the OKD cluster. There are several log aggregation tools in the market that offer similar capabilities. This guide describes the process of deploying a log aggregating tool, **EFK (Elasticsearch, FluentD, and Kibana)**, using the ansible-playbook script provided by the OKD.
```
Comment (t-koman) [inline README.adoc]: 17) --->change phrase --deploy **a a** log aggregation tool-- TO --deploy a log aggregation tool--

Last sentence ---- change --This guide describes the process of deploying a log aggregating tool -- TO -- This guide describes the process of deploying a log **aggregation** tool --

File: README.adoc [hunk 2]
```diff
@@ -14,9 +14,9 @@ permalink: /guides/app-logging/
...
 
-OKD provides a preconfigured EFK(Elasticsearch, FluentD, and Kibana) stack, which a DevOps team uses to aggregate all container logs. You can install EFK stacks by using the ansible-playbook that resides in the https://github.com/openshift/openshift-ansible/tree/release-3.11[openshift-ansible] repository. After the installation completes, the EFK deployments reside inside the *openshift-logging* namespace of the OKD cluster.
+You use this preconfigured EFK stack to aggregate all container logs and use the the ansible-playbook that resides in the https://github.com/openshift/openshift-ansible/tree/release-3.11[openshift-ansible] repository to install EFK stacks on a OKD cluster. After a successful installation, the EFK deployments reside inside the *openshift-logging* namespace of the OKD cluster.
```
Comment (t-koman) [inline README.adoc]: 19) First sentence - change phrase -- and use **the the** ansible-playbook -- TO -- and use the ansible-playbook --

**Review by t-koman — APPROVED** [review]
@fwji Frank, fix what I noted  - I approve the update.   Any questions, let me know.....

File: README.adoc [hunk 3]
```diff
@@ -14,7 +14,9 @@ permalink: /guides/app-logging/
...
 
-OKD (Openshift Origin Community Distribution) provides a preconfigured EFK(Elasticsearch, FluentD, and Kibana) stack, which DevOps uses to aggregate all container logs. You can install EFK stacks by using the ansible-playbook command that resides in the https://github.com/openshift/openshift-ansible/tree/release-3.11[openshift-ansible] repository. After the installation completes, the EFK deployments reside inside the *openshift-logging* namespace of the OKD cluster.
+A Kubernetes environment constantly produce logs by its running pod processes. To effectively manage this log data and ensure no loss of log data occurs when a pod terminates, deploy a a log aggregation tool on the user's OKD cluster to control the logging activities. Log aggregation tools help users persist, search, and visualize the log data that is gathered from the pods across the OKD cluster. There are several log aggregation tools in the market that offer similar capabilities. This guide describes the process of deploying a log aggregating tool, **EFK (Elasticsearch, FluentD, and Kibana)**, using the ansible-playbook script provided by the OKD.
```
Comment (donbourne) [inline README.adoc]: provided by the OKD -> provided by OKD

Comment (t-koman) [reply README.adoc]: Don - I approve your change, but earlier we have a sentence that reads.....  ,deploy **a a** log aggregation tool on the user's OKD cluster.....  CHANGE to deploy a log aggregation tool on the user's OKD cluster....         (..... only need one "a")

File: README.adoc (hunk 1 above)
Comment (t-koman) [reply README.adoc]: @fwji @donbourne Don, I approve your change, but we still have that sentence that says..... deploy a a log aggregation tool on the user's OKD cluster......  (need to drop one "a"...)

Comment (fwji) [reply README.adoc]: 1. removed extra 'a'
2. 'aggregating' to 'aggregation'

**Review by fwji — COMMENTED** [review]
Commit with Don and Tom's suggested fix.

**Pull request merged**